massgov/mayflower-patternlab
# DP-7063 Added SVG Chat icon

## Description
The Decision Tree disclaimer calls for a new icon from the existing library. This implements it so it can be made available everywhere.

## Related Issue / Ticket
https://jira.state.ma.us/browse/DP-7063

## Steps to Test
1. Compile Mayflower branch and view in browser
2. Check template at Atoms > Icons > SVG Chat
3. Confirm that a speech bubble icon is displayed, matching the screenshot below

## Screenshots
<img width="85" src="https://user-images.githubusercontent.com/18170074/34500357-638babb8-efd8-11e7-8b5a-43d2c02be8b4.png">

## Additional Notes:
Once this is made available to Mass.gov, the decision tree template can be adjusted to include the SVG Chat template instead of manually including a copy of the markup as it does now.

#### Impacted Areas in Application
N/A

#### @TODO
N/A

#### Today I learned...
N/A
